src/ui/components/ClientsGrid/component.ts

```diff
@@ -127,6 +127,13 @@ const allClients = [
     text:
       'ERDiL builds natural language processing software that helps companies analyze messages from their customers. They reached out to simplabs for guidance on writing tests for their Ember.js based dashboard app and establishing a sustainable testing culture.',
   },
+  {
+    id: 'poellath',
+    name: 'POELLATH',
+    src: '/assets/images/logos/poellath.png',
+    text:
+      'POELLATH is a german law firm specializing in advice on transactions and asset management. When they were looking to build custom software tools to improve workflows, we conducted a product strategy workshop with them to develop a clear understanding of these tools and enable next steps.',
```

Comment:
```suggestion
      'POELLATH is a German law firm specializing in advice on transactions and asset management. When they were looking to build custom software tools to improve workflows, we conducted a product strategy workshop with them to develop a clear understanding of these tools and enable next steps.',
```

Gentillics in English are capitalized.